content/en/tracing/serverless_functions/_index.md

```diff
@@ -0,0 +1,222 @@
+var http = require('http');
+```
+
+To instrument PostgreSQL queries:
+
+```js
+var AWSXRay = require('aws-xray-sdk');
+var pg = AWSXRay.capturePostgres(require('pg'));
+var client = new pg.Client();
+```
+
+To instrument MySQL queries:
+
+```js
+var AWSXRay = require('aws-xray-sdk');
+var mysql = AWSXRay.captureMySQL(require('mysql'));
+//...
+var connection = mysql.createConnection(config);
+```
+
+For further configuration, creating subsegments, and recording annotations, consult the [X-Ray Node.js docs][1].
+
+
+[1]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-nodejs.html
+{{% /tab %}}
+{{% tab "Python" %}}
+
+Install the X-Ray tracing library:
+
+```bash
+pip install aws-xray-sdk
+```
+
+To patch [all libraries][1] by default, add the following to the file containing your Lambda handlers:
+
+```python
+from aws_xray_sdk.core import xray_recorder
+from aws_xray_sdk.core import patch_all
+patch_all()
+```
+
+Note that tracing `aiohttp` requires [specific instrumentation][2].
+
+For further configuration, creating subsegments, and recording annotations, consult the [X-Ray Python docs][3].
+
+
+[1]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-python-patching.html
+[2]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-python-httpclients.html
+[3]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-python.html
+{{% /tab %}}
+{{% tab "Go, Ruby, Java, .NET" %}}
+
+For all other runtimes, consult the X-Ray SDK docs:
+
+- [X-Ray SDK for Go][1]
+- [X-Ray SDK for Ruby][2]
+- [X-Ray SDK for Java][3]
+- [X-Ray SDK for .NET & Core][4]
+
+
+[1]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-go.html
+[2]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-ruby.html
+[3]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-java.html
+[4]: https://docs.aws.amazon.com/en_pv/xray/latest/devguide/xray-sdk-dotnet.html
+{{% /tab %}}
+{{< /tabs >}}
+
+### Data Collected
+
+The AWS X-Ray integration pulls in trace data from AWS, and does not collect any metrics or logs.
+
+## Tracing across AWS Lambda and hosts
+
+When applicable, Datadog merges AWS X-Ray traces with native Datadog APM traces. This means that your traces will show the complete picture of requests that cross infrastructure boundaries, whether it be AWS Lambda, containers, on-prem hosts, or managed services.
+
+1. Enable the [AWS X-Ray integration][13] for tracing your Lambda functions.
+2. Add the [Datadog Lambda Library](#Datadog-Lambda-Library) to your Lambda functions.
+3. [Set up Datadog APM][14] on your hosts and container-based infrastructure.
+
+**Note**: For X-Ray and Datadog APM traces to appear in the same flame graph, all services must have the [same `env` tag](#the-env-tag).
+
+{{< img src="integrations/amazon_lambda/lambda_host_trace.png" alt="trace of a request from a host to a Lambda function" >}}
+
+### Organizing your infrastructure with tags
```

Comment:
I'm not sure these instructions are the best practice for tagging your functions with X-Ray? What do you think?

Reply:
FWIW we're looking to provide more information on tagging on the serverless side.